.eslintrc.json

```diff
@@ -7,7 +7,15 @@
     ],
     "parserOptions": {
         "ecmaVersion": 6,
-        "sourceType": "module"
+        "sourceType": "module",
+        "project": [
+            "./tsconfig.json",
+            "./tsconfig-eslint.json",
```

Comment:
Now I see this error in the editor:

> Parsing error: Cannot read file '/workspace/langium/tsconfig-eslint.json'.

Reply:
Yes, this is kind of a nightmare. I found any way to get everything (IDE, CLI and Tests) working at once. 
Is it possible to move the `test` folder inside the `src` and then exclude it from packaging?